Swiftfin/Components/FilterDrawerHStack/FilterDrawerHStack.swift

```diff
@@ -7,18 +7,22 @@
 //
 
 import JellyfinAPI
+import Defaults
 import SwiftUI
 
 struct FilterDrawerHStack: View {
 
+    @Default(.Customization.Filters.filterDrawerButtons)
+    private var filterActiveDrawerButtons
+    
     @ObservedObject
     private var viewModel: FilterViewModel
 
     private var onSelect: (FilterCoordinator.Parameters) -> Void
 
     var body: some View {
         HStack {
-            if viewModel.currentFilters.hasFilters {
+            if !filterActiveDrawerButtons.isEmpty && viewModel.currentFilters.hasFilters {
```

Comment:
We don't want to disable the view of the buttons if there are none here - we want to disable the viewing of the `FilterDrawerHStack` view itself. This would mean that the drawer itself isn't shown when used in the `navBarDrawer` `View` extension.

Reply:
In iOS, I found 2 references to this FilterDrawerHStack in libraryView and searchView. Is the goal something like this:

<details>
<summary>Disable ScrollView Inside navBarDrawer</summary>

<img src="https://github.com/jellyfin/Swiftfin/assets/37599297/9cfea651-842b-40f7-b062-9da9193af75e" width="30%" height="30%">
</details>

Since it looks like the FilterDrawerHStack is passed into the navBarDrawer, I wasn't sure how to handle this aside from removing the FilterDrawer inside. Let me know if there is a better route than the screenshot above!

Reply:
See previous screenshot. Called the searchFilterDrawerButton setting for SearchView and libraryFilterDrawerButton for libraryView. Validated that this removes this the filters from the NavBar. It works the same as before where there is still a NavBar but it's empty. This is in the PR now but I think it's valid to want to approach this from another route if there is a way for me to wrap this where the navbar doesn't exist at all opposed to just being empty. I was wrapping the navbar in the If/Then before but it through me a View error.